intranet/apps/dataimport/management/commands/import_staff.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import sys
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from ....users.models import Email
+
+
+class Command(BaseCommand):
+    help = (
+        "Given an CSV of staff, add them to Ion. Required columns include 'Username', 'First Name', 'Last Name', "
+        "and 'Middle Name'. Optional columns include 'Nick Name' and 'Gender'."
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("--filename", dest="filename", type=str, required=True, help="Filename to import data from. Required.")
+        parser.add_argument("--run", action="store_true", dest="run", default=False, help="Actually run.")
+        parser.add_argument("--confirm", action="store_true", dest="confirm", default=False, help="Skip confirmation.")
+
+    def ask(self, q):
+        if input(f"{q} [Yy]: ").lower() != "y":
+            self.stdout.write(self.style.ERROR("Abort."))
+            sys.exit()
+
+    def handle(self, *args, **options):
+        # Read the data file
+        with open(options["filename"], "r") as csvfile:
+            data = list(csv.DictReader(csvfile))
+
+        do_run = options["run"]
+        if do_run:
+            if not options["confirm"]:
+                self.ask("Add new users?")
+        else:
+            self.stdout.write(self.style.WARNING("In pretend mode."))
+
+        # Loop through our new users
+        for new_user in data:
+            nickname = new_user["Nick Name"].strip() if "Nick Name" in new_user.keys() else ""
+            gender = new_user["Gender"].strip() == "M" if "Gender" in new_user.keys() else None  # TODO: is it "M" or "Male" or something else?
```

Comment:
I don't know that we record gender currently...most users don't seem to have it.

Reply:
True, but it doesn't hurt - either way, if we don't supply a gender column, it would be entered as `None`.

The two users I'm importing I'm told were female (the email mentions them as "Ms. {{ NAME }}") so we may as well put that in while we're at it